product/urls.py

```diff
@@ -1,5 +1,6 @@
-from django.urls import path
+from django.urls   import path
+from .views        import ProductDetailView,
 
 urlpatterns = [
-
+    path('/product-detail/<int:product_id>', ProductDetailView.as_view()),
```

Comment:
다음과 같이 urlpattern을 설정하면 product detail를 보내주는 API의 urlpattern이 product/product-detail/<product_id>가 됩니다.
```suggestion
    path('/<int:product_id>', ProductDetailView.as_view()),
```
위와 같이 수정하면 /product/<product_id> 만으로도 제품들의 정보를 받아 올 수 있고 더 RESTfull해 집니다.